frontend/ui/widget/filechooser.lua

```diff
@@ -282,13 +277,12 @@ function FileChooser:genItemTableFromPath(path)
 
     local item_table = {}
     for i, dir in ipairs(dirs) do
-        -- count sume of directories and files inside dir
+        -- count number of folders and files inside dir
         local sub_dirs = {}
         local dir_files = {}
         local subdir_path = self.path.."/"..dir.name
         self.list(subdir_path, sub_dirs, dir_files, true)
-        local num_items = #sub_dirs + #dir_files
-        local istr = ffiUtil.template(N_("1 item", "%1 items", num_items), num_items)
+        local istr = T("%1 • %2", #sub_dirs, #dir_files)
```

Comment:
Not fond of this display, `3 • 4`, I wouldn't know if the first number are files or subdirs - or even that's what these two numbers means :/
`3f + 4d`
`3 + 4/`
`4/ + 3` (if subsubdirs are displayed first when we walk in this subdir)
or some small unicode or nerdfont symbols for files or folders.
Although I don't know how essential it is to show these 2 numbers instead of just one, as mostly everything else does (but I'd like if we show more/differently, but it has to be intuitive)/